noxfile.py

```diff
@@ -39,7 +39,7 @@
 
 DEFAULT_PYTHON_VERSION = "3.9"
 SYSTEM_TEST_PYTHON_VERSIONS = ["3.9", "3.11", "3.12", "3.13"]
-UNIT_TEST_PYTHON_VERSIONS = ["3.9", "3.11", "3.12", "3.13"]
+UNIT_TEST_PYTHON_VERSIONS = ["3.9", "3.11", "3.12", "3.13", "3.14"]
```

Comment:
Why is 3.10 missing (even before this PR)? Either we test on all supported versions, or we sample fewer (like the lowest and highest), don't you think?